### PR TITLE
ide-usage.md: Eclipse section assumes JDK 1.7 default #7110

### LINKED ISSUE #7110: ide-usage.md: Eclipse section assumes JDK 1.7 default
Even though there is a note that states this is not required, the project fails to build when following the steps exactly if JDK 1.7 is not the default. To fix, an additional step is required:

`Project properties -> Java Compiler -> Tick "Use compliance from execution environment 'JavaSE-1.7' on the 'Java Build Path'"`

Add this additional step with an accompanying screenshot (~~preferably macOS for consistency with the other screenshots~~ use the below screenshot) to `ide-usage.md`.

### PR DESCRIPTION
**Pull request by VamsiSangam**
Fixes #7110 

Corrected docs as per the issue description. [Demo](https://github.com/VamsiSangam/teammates/blob/7110-ide-usage-correction/docs/ide-usage.md)

### CODE REVIEW

**Comment by chowyb** [comment]
Quick comment: the image should be cropped (similar to the other images on the document)

**Comment by VamsiSangam** [comment]
Sure, will do.

**Comment by VamsiSangam** [comment]
Ready for review.

File: docs/ide-usage.md [hunk 1]
```diff
@@ -32,7 +32,9 @@ The following plugins are needed:
 
      ![setupguide-3.png](images/setupguide-3.png)
 
-   Note that none of the App Engine SDK or JDK to be used are required to be the `default`.
+   * Set Java SE 7 as default JDK. To do this, go to ``` Project properties → Java Compiler → Tick "Use compliance from execution environment 'JavaSE-1.7' on the 'Java Build Path'" ```.
```
Comment (wkurniawan07) [inline docs/ide-usage.md]: This is a wrong fix. The statement that none are required to be the `default` still holds.

What needs to be done here needs to be under "Configure the following project-specific settings (all can be found in `Project → Properties → ...`)" instead.

Also, on top of the `Java Compiler` setting, one should also ensure that the `Java Build Path` is set to JDK 7, as shown in the issue thread.

**Comment by VamsiSangam** [comment]
Made corrections. Please review.

File: docs/ide-usage.md [hunk 2]
```diff
@@ -53,6 +53,14 @@ The following plugins are needed:
      * ORM Enhancement: `App Engine → ORM` → clear all the entries, and add the following entry: `src/main/java/teammates/storage/entity/*.java`.
      * Validation exclusion: `App Engine → Validation` → add two entries: `src/test/java` and `src/client/java`.
      * WAR directory: `Web Application` → tick both `This project has a WAR directory` and `Launch and deploy from this directory`, and enter `src/main/webapp` as `WAR directory`.
+   * Java Compiler: Go to `Project → Properties → Java Compiler` and  tick `"Use compliance from execution environment 'JavaSE-1.7' on the 'Java Build Path'" `.
```
Comment (wkurniawan07) [inline docs/ide-usage.md]: * JDK: `Java Build Path → Libraries` → ensure that the system library used is JDK 7.
* Compiler compliance: `Java Compiler` → tick `Use compliance from execution environment 'JavaSE-1.7' on the 'Java Build Path'`

The order needs to be that way as the build path needs to be set first before compiler compliance.

Also, the image for compiler compliance looks off (different resolution from the rest). Use this instead:
screen shot 2017-04-14 at 9 48 13 pm

File: docs/ide-usage.md [hunk 3]
```diff
@@ -32,7 +32,7 @@ The following plugins are needed:
 
      ![setupguide-3.png](images/setupguide-3.png)
 
-   Note that none of the App Engine SDK or JDK to be used are required to be the `default`.
+     Note that none of the App Engine SDK or JDK to be used are required to be the `default`.
```
Comment (wkurniawan07) [inline docs/ide-usage.md]: Revert this. It may look trivial, but in markdown syntax, this changes the indentation level.

**Comment by VamsiSangam** [comment]
Ready for review.

File: docs/ide-usage.md [hunk 4]
```diff
@@ -53,6 +53,14 @@ The following plugins are needed:
      * ORM Enhancement: `App Engine → ORM` → clear all the entries, and add the following entry: `src/main/java/teammates/storage/entity/*.java`.
      * Validation exclusion: `App Engine → Validation` → add two entries: `src/test/java` and `src/client/java`.
      * WAR directory: `Web Application` → tick both `This project has a WAR directory` and `Launch and deploy from this directory`, and enter `src/main/webapp` as `WAR directory`.
+   * JDK: `Java Build Path → Libraries` → ensure that the system library used is JDK 7.
+
+       ![setupguide-7.png](images/setupguide-7.png)
+
+   * Compiler compliance: `Java Compiler` → tick `"Use compliance from execution environment 'JavaSE-1.7' on the 'Java Build Path'" `.
```
Comment (wkurniawan07) [inline docs/ide-usage.md]: Remove the double quotes (`"`). Should just be: tick `Use compliance ... Build Path'`.

**Comment by VamsiSangam** [comment]
Made requested changes. Please review.

File: docs/ide-usage.md [hunk 5]
```diff
@@ -53,6 +53,14 @@ The following plugins are needed:
      * ORM Enhancement: `App Engine → ORM` → clear all the entries, and add the following entry: `src/main/java/teammates/storage/entity/*.java`.
      * Validation exclusion: `App Engine → Validation` → add two entries: `src/test/java` and `src/client/java`.
      * WAR directory: `Web Application` → tick both `This project has a WAR directory` and `Launch and deploy from this directory`, and enter `src/main/webapp` as `WAR directory`.
+   * JDK: `Java Build Path → Libraries` → ensure that the system library used is JDK 7.
+
+       ![setupguide-7.png](images/setupguide-7.png)
```
Comment (whipermr5) [inline docs/ide-usage.md]: Can this screenshot show the the top of the window for consistency with the other screenshots (i.e. only crop the bottom)?

File: docs/ide-usage.md [hunk 6]
```diff
@@ -53,6 +53,14 @@ The following plugins are needed:
...
+
+   * Compiler compliance: `Java Compiler` → tick `Use compliance from execution environment 'JavaSE-1.7' on the 'Java Build Path'`.
+
+       ![setupguide-8.png](images/setupguide-8.png)
```
Comment (whipermr5) [inline docs/ide-usage.md]: Crop the left border off for this screenshot, just like the other screenshots.

Comment (VamsiSangam) [inline docs/ide-usage.md]: Done!

File: docs/ide-usage.md (hunk 5 above)
Comment (VamsiSangam) [inline docs/ide-usage.md]: I don't have an iOS machine. Could you please provide me with the screenshot?

Comment (VamsiSangam) [inline docs/ide-usage.md]: FYI, the image used here isn't taken from the issue, @wkurniawan07 gave me another image saying the [former had resolution issues](https://github.com/TEAMMATES/teammates/pull/7123#discussion_r111604451).

Comment (whipermr5) [inline docs/ide-usage.md]: Oops, apologies for the oversight 😅 I managed to use a custom resolution setting through a third party tool to get a screenshot of the same resolution but alas, I realised that screenshot actually contains our home directory username:
screen shot 2017-04-17 at 1 04 45 am

Hence, it'd be best to use @wkurniawan07's for consistency. @wkurniawan07 could you please provide a screenshot 😅

Comment (wkurniawan07) [inline docs/ide-usage.md]: Apologies for that oversight on my part 😅 here's the required screenshot:

screen shot 2017-04-19 at 12 33 51 am

**Comment by VamsiSangam** [comment]
Ready for review.

**Pull request merged**